src/ethereum_test_fixtures/state.py

```diff
@@ -98,6 +98,8 @@ class FixtureConfig(CamelModel):
     """Chain configuration for a fixture."""
 
     blob_schedule: FixtureBlobSchedule | None = None
+    chain_id: int = Field(1, alias="chainid")
```

Comment:
Same comment as in the Blockchain test format, although here the `FixtureForkPostIndexes` are actually `int`, so should not be as problematic as in the other one.